src/pageObject.js

```diff
@@ -1,15 +1,39 @@
 'use strict';
 
 class Header {
-
+  get logo() {
+    return `Conduit logo`;
+  }
 };
 
 class PageObject {
+  constructor(url) {
+    this.url = url;
+    this.header = new Header();
+  }
 
+  clickOnLogo() {
+    return `Click on the Conduit logo`;
```

Comment:
The method should return a string 'Click on the' + logo from the `Header` class. Use interpolation and `logo` method from `Header` class